components/org.wso2.carbon.identity.api.server.authenticators/org.wso2.carbon.identity.api.server.authenticators.v1/pom.xml

```diff
@@ -169,6 +164,12 @@
             <groupId>commons-beanutils</groupId>
             <artifactId>commons-beanutils</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.identity.server.api</groupId>
+            <artifactId>org.wso2.carbon.identity.api.server.authenticators.common</artifactId>
```

Comment:
why are we adding the dependency again? The same dependency is already added https://github.com/wso2/identity-api-server/pull/688/files#diff-c6a1f5353ad37692ea971d36c27f96a68484a2621882c1b4edaf9636bc2e2694R155